query-connector/src/app/queryBuilding/components/SelectionTable.tsx

```diff
@@ -56,13 +60,13 @@ export const SelectionTable: React.FC<SelectionTableProps> = ({
   };
 
   const handleSingleCheckboxToggle = (
-    valueSetType: DibbsValueSetType,
-    groupedValueSet: GroupedValueSet,
+    activeConceptType: DibbsConceptType,
+    groupedValueSet: VsGrouping,
     isBatchUpdate: boolean = false,
     batchValue?: boolean,
   ) => {
     const key = `${groupedValueSet.valueSetName}:${groupedValueSet.author}:${groupedValueSet.system}`;
-    const updatedVS = groupedValueSetsForCondition[valueSetType][key];
+    const updatedVS = groupedValueSetsForCondition[activeConceptType][key];
```

Comment:
a bit confused by the naming of this to `activeConceptType`. Aren't these valueset-related, not concept-related?

Reply:
good point! went ahead and changed this back